src/api/ratelimits.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Rate Limits
+sidebar_order: 5
+---
+
+Sentry rate limits every API request made to prevent abuse and resource overuse. The limit is applied to each caller per endpoint.
+
+The rate limit follows a fixed window approach. Requests are counted into time buckets, determined by the window size. If the number of requests from a caller exceeds the number of allowed requests within a window, then that request will be rejected.
+
+While there is a default value, each endpoint can have their own maximum number of requests and window size. You can track your rate limit usage by looking at special headers in the response
+
+## Headers
+
+Every API request response will include the following headers:
+
+- `X-Sentry-Rate-Limit-Limit`
+  - The maximum number of requests allowed within the window
+- `X-Sentry-Rate-Limit-Remaining`
+  - The number of requests this caller has left on this endpoint within the current window
+- `X-Sentry-Rate-Limit-Reset`
+  - The time when the next rate limit window begins and the count resets
+  - Measured in UTC seconds from epoch
```

Comment:
To make it fit the set pattern a bit better. If this section grows in the future, then it's fine to break it back out. 
```suggestion
  - The time when the next rate limit window begins and the count resets, measured in UTC seconds from epoch
```